packages/common/src/utils/fileUtils.ts

```diff
@@ -1,3 +1,4 @@
+import firstline from 'firstline';
```

Comment:
I see nothing wrong with this module but is it really needed to read one line of a file?

Reply:
this is to validate if the .tsv contains required columns. so not need to read entire .tsv